src/applications/vaos/tests/appointment-list/components/UpcomingAppointmentsList.unit.spec.jsx

```diff
@@ -628,7 +640,7 @@ describe('VAOS <UpcomingAppointmentsList>', () => {
     });
 
     await screen.findByText(
-      new RegExp(startDate.tz('America/New_York').format('dddd, MMMM D'), 'i'),
+      new RegExp(startDate.tz('America/Denver').format('dddd, MMMM D'), 'i'),
```

Comment:
The new logic to set the date when these tests run revealed this issue, where we were adjusting for a timezone that did not match the facility.